docs/schema/guidance/record-ids.rst

```diff
@@ -4,4 +4,53 @@
 Record identifiers
 ==================
 
-[Stub page]
+To publish beneficial ownership data in BODS format, an information management system MUST maintain record identifiers for each element (person, entity and relationship) in each beneficial ownership network.
+
+Record identifiers MUST be unique within the publisher's system and never reused to refer to a different element. 
+
+There are situations where a publisher's system can allow information about the same person, entity or relationship to be collected in relation to the beneficial ownership networks of two (or more) different subjects. In these cases, information from different sources about the same person, entity or relationship can exist side-by-side and needs to be represented in published data. In these cases, each set of information MUST have its own record identifier. (See the example below.)
+
+The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person in question, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.  
+
+The identity of a relationship is constituted by the identities of its subject and interested party. Therefore the value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
```

Comment:
I think this lead in sentence can be deleted. 
```suggestion
The value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
```

Reply:
I'm going to leave this in since it's a foundational principle that determines when a relationship record should and shouldn't be updated.